src/components/SideNavigation/SideNavigation.tsx

```diff
@@ -107,27 +108,45 @@ class SideNavigation extends React.PureComponent<Props, State> {
       const childrenMarkup = full.children !== undefined || null ? full.children.map((child: any, index: number) => {
         return (
           <div key={index}>
-            <a className={childLiClass} onClick={child.action} aria-disabled={false} id={componentId ? `${componentId}${child.label}` : ''}>
-              <Icon source={child.icon} componentColor="white" componentClass={theme.customIcon} theme={theme} />
-              {child.label}
-            </a>
+            { child.notActionable ?
+              (
+                <div className={childLiClass} style={{ cursor: 'default' }} aria-disabled={false} id={componentId ? `${componentId}${child.label}` : ''}>
+                  <Icon source={child.icon} componentColor="white" componentClass={theme.customIcon} theme={theme} />
+                  {child.label}
+                </div>
+              ) : (
+                <Button componentSize="slim" componentClass={childLiClass} onClick={child.action} aria-disabled={false} componentId={componentId ? `${componentId}${child.label}` : ''} plain fullWidth >
+                  <Icon source={child.icon} componentColor={'black'} componentClass={theme.customIcon} theme={theme} />
```

Comment:
@sumeet-jadhav the icons color should not be "black"

Reply:
Done